tracer/src/Datadog.Trace.Tools.dd_dotnet.SourceGenerators/Datadog.Trace.Tools.dd_dotnet.SourceGenerators.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <IsRoslynComponent>true</IsRoslynComponent>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!--  Using the root namespace so generators are all in the root + we can control how they're written to disk  -->
+    <RootNamespace></RootNamespace> 
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
+    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.3.0" />
```

Comment:
Do we need those? 🤔

Reply:
Oops, no, probably not. That's probably related to the messagepack stuff